construct-sturdy-uis-with-xstate/notes/01-introducing-construct-sturdy-uis-with-xstate.md

```diff
@@ -0,0 +1,40 @@
+# Introducing Construct Sturdy UIs with Xstate
+
+- [Video Link](https://egghead.io/lessons/react-introducing-construct-sturdy-uis-with-xstate)
+
+### 3 main reasons why `Xstate` is awesome
+
+- State machines encourage planning out component before you created.
+  - Planned code is better code.
+- State machines are an old idea and this means that it has been battletested.
+- Xstate provides a browser visualizer in which we can insert our coud and it will create diagrams. This helps communicate the idea better.
```

Comment:
```suggestion
- Xstate provides a browser visualizer in which we can insert our code and it will create diagrams. This helps communicate the idea better.
```